lib/src/map_widget.dart

```diff
@@ -87,7 +87,10 @@ class MapWidget extends StatefulWidget {
   /// The Initial Camera options when creating a MapWidget.
   final CameraOptions? cameraOptions;
 
-  /// Flag indicating to use a TextureView as render surface for the MapWidget. Default is false. Only works for Android.
+  /// Flag indicating to use a TextureView as render surface for the MapWidget.
+  /// Only works for Android.
+  /// FIXME Flutter 3.x has memory leak on Android using in SurfaceView mode, see https://github.com/flutter/flutter/issues/118384
+  /// As a workaround default is true.
```

Comment:
is this whole comment more relevant in L19?

Reply:
Added there as well